internal/util/cursor/cursor.go

```diff
@@ -12,7 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package controlplane
+// Package cursor provides a way to encode and decode cursors for paginated queries
+package cursor
```

Comment:
Are the cursors ever used outside of the control plane? If not, I think it may make sense to keep them as a sub-package of controlplane.

Reply:
Yes, they should probably go into a less-generic package than `util`, but that is something we can worry about when you introduce your next PR(s). I'm happy to leave them in util for now.